healvis/beam_model.py

```diff
@@ -278,6 +281,8 @@ def __init__(self, beam_type, gauss_width=None, diameter=None):
         Args:
             beam_type : str or callable, type of beam to use. options=['uniform', 'gaussian', 'airy', callable]
             gauss_width : float, standard deviation [degrees] for gaussian beam
+            spectral_index: (float, optional) Scale gaussian beam width as a power law with frequency.
+                                               With this, gauss_width indicates the width at the lowest frequency.
```

Comment:
I believe doc-string indent is a single indent, and doesn't follow the same line-continuation style as real code. see other doc strings for examples